.storybook/2.DOCUMENTATION.stories.mdx

```diff
@@ -59,7 +59,7 @@ export default {
   // The `title` effects the components tile and location in storybook
   // It should follow the folder structure location of the component. Don't use spaces.
   title: 'Components/UI/Button',
-  id: __filename, // The file name id is used to track what storybook files have changed in CI
+ // The file name id is used to track what storybook files have changed in CI
```

Comment:
I think we can probably remove this comment altogether?
```suggestion
```

Reply:
When #17092 is merged it will change all storybook URLs and will essentially break any link references to those stories. We will need to update internal and external links. This ticket is to update all internal links inside of the repo.